main.py

```diff
@@ -5,12 +5,7 @@
 from xml.etree import ElementTree as etree
 from urllib.parse import urlencode
 import json
-import os
-
-CONSUMER_KEY = os.getenv("CONSUMER_KEY")
-CONSUMER_SECRET = os.getenv("CONSUMER_SECRET")
-ACCESS_KEY = os.getenv("ACCESS_KEY")
-ACCESS_SECRET = os.getenv("ACCESS_SECRET")
+from environment_variables import *
```

Comment:
Using *-imports is heavily discouraged: you can't at a glance understand from the script which names are imported, and if you import * from 2+ modules, it becomes a nightmare. Just list all imports by name.

Reply:
I thought it's so cool not to write this long line of names,  but I see your point.